impl_core/src/main/java/io/opencensus/trace/export/ExportComponentImpl.java

```diff
@@ -42,9 +42,14 @@ public SampledSpanStore getSampledSpanStore() {
     return null;
   }
 
-  /** Constructs a new {@code ExportComponentImpl}. */
-  public ExportComponentImpl() {
+  /**
+   * Constructs a new {@code ExportComponentImpl}.
+   *
+   * @param supportInProcessStores {@code true} to instantiate {@link ActiveSpansExporter} and
+   *     {@link SampledSpanStore}.
+   */
+  public ExportComponentImpl(boolean supportInProcessStores) {
```

Comment:
Suggestion: Using separate factory methods for full and lite functionality (or using an enum instead of boolean) would make the calling code easier to read.